File: 백준/Silver/1929. 소수 구하기/소수 구하기.py
M, N = map(int, input().split())

for i in range(M, N + 1):
    if i == 1:
        continue
    # 두수의 곱은 대칭이므로 제곱근까지만 확인하면 된다.
    for j in range(2, int(i ** 0.5) + 1):
        if i % j == 0:
            break
    else:
        print(i)

File: 백준/Silver/1929. 소수 구하기/소수 구하기.js
const fs = require("fs");
const [M, N] = fs
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split(" ")
  .map(Number);

for (let i = M; i <= N; i++) {
  if (i === 1) continue;

  let isPrime = true;

  for (let j = 2; j <= Math.sqrt(i); j++) {
    if (i % j === 0) {
      isPrime = false;
      break;
    }
  }

  if (isPrime) console.log(i);
}
